packages/gatsby-theme-vtex/src/components/Minicart/Drawer/Quantity.tsx

```diff
@@ -0,0 +1,54 @@
+import { useIntl } from '@vtex/gatsby-plugin-i18n'
+import { Input, Select } from '@vtex/store-ui'
+import React, { useState } from 'react'
+
+import { useItem } from './useItem'
+import { useUpdateItems } from './useUpdateItems'
+
+type Props = {
+  index: number
+  variant?: string
+}
+
+const MinicartQuantity = (props: Props) => {
```

Comment:
I think we could add a loading indicator in here as well so the user knows when it's updating a cart item

Reply:
where would the loader be?